source/camera/info_util_other/warm-up-sample.py

```diff
@@ -0,0 +1,54 @@
+"""
+This example shows how to warm up the camera. 
+"""
+
+import zivid
+from time import sleep
+from datetime import datetime, timedelta
+
+
+def _main():
+    app = zivid.Application()
+
+    print("Connecting to camera")
+    camera = app.connect_camera()
+
+    warmup_time = timedelta(minutes=10)
+    capture_cycle = timedelta(seconds=5)
+    max_capture_time = timedelta(milliseconds=1000)
+
+    print("Getting camera settings")
+    suggest_settings_parameters = zivid.capture_assistant.SuggestSettingsParameters(
+        max_capture_time=max_capture_time,
+        ambient_light_frequency=zivid.capture_assistant.SuggestSettingsParameters.AmbientLightFrequency.none,
+    )
+    settings = zivid.capture_assistant.suggest_settings(camera, suggest_settings_parameters)
+
+    before_warmup = datetime.now()
+
+    print("Starting warm up")
```

Comment:
An we are not printing "Starting warm up" in c++. Should be consistent. 
Maybe this and the print above can be merged if we want to print both. 
Starting warm up, total x minutes?
If we only want to keep one I think "starting" is nice than the 10 minutes. But no strong opinions here